ProcessLib/HeatConduction/Tests.cmake

```diff
@@ -42,3 +42,15 @@ AddTest(
 #      EXECUTABLE ogs
 #      EXECUTABLE_ARGS wedge_1e2_axi_ang_0.02.prj
 # )
+# The 25 BHE array benchmark
+# test results are compared to 2D simulation result
+AddTest(
+        NAME BHE_Array_2D
+        PATH Parabolic/T/2D_BHE_array
+        EXECUTABLE ogs
+        EXECUTABLE_ARGS bhe2d.prj
+        TESTER vtkdiff
+        DIFF_DATA
+        standard_solution_bhe2d_pcs_0_ts_840_t_72576000.000000.vtu bhe2d_pcs_0_ts_840_t_72576000.000000.vtu temperature temperature 1e-4 1e-4
```

Comment:
Not sure if the absolute and relative errors of 1e-4 aren't too sloppy. I'd expect something in range 1e-10 to 1e-15 for a benchmark. Of course, if temperature is crossing 0, then the relative error does not make sense and should be set to 0.